tsdef/twilio-video.d.ts

```diff
@@ -0,0 +1,13 @@
+import { ConnectOptions, CreateLocalTrackOptions, CreateLocalTracksOptions, LocalTrack } from './types';
```

Comment:
Why is this file necessary?

Reply:
I included these imports since they're part of the top level exports options.